code_generation/producers/event.py

```diff
@@ -46,3 +46,54 @@
     scopes=["global"],
     subproducers=[DiElectronVeto, DiMuonVeto],
 )
+
+GGH_NNLO_Reweighting = Producer(
+    name="GGH_NNLO_Reweighting",
+    call='htxs::ggHNLLOWeights({df}, {output}, "{ggHNNLOweightsRootfile}", "{ggH_generator}", {input})',
+    input=[nanoAOD.HTXS_Higgs_pt, nanoAOD.HTXS_njets30],
+    output=[q.ggh_NNLO_weight],
+    scopes=["global", "em", "et", "mt", "tt"],
```

Comment:
right, I think it can either be used in global or in all other channels in case one wants to have certain filters run first. Therefore, I'd allow everything here.